src/index.html

```diff
@@ -9,6 +9,66 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home" class="header__a">
+        <img 
+          src="images/logo.png" 
+          alt="moyo-logo" 
+          class="header__img">
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__li">
+            <a href="#apple" class="is-active nav__a">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a href="#samsung" class="nav__a">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__li">
+            <a href="#smartphones" class="nav__a">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__li">
+            <a 
+              href="#laptops & computers" 
+              data-qa="hover" 
+              class="nav__a">
+              Laptops & computers
```

Comment:
```suggestion
              class="nav__a"
            >
              Laptops & computers
```

Reply:
![6](https://user-images.githubusercontent.com/117728583/220600149-b47e2392-e1c1-4252-9617-95b5ae482da4.png)
Hello! Firstly I did it, than another mentor told me to fix it, like on the photo. Is it the same or not? And should I do?

Reply:
here as well